src/Node/HTTP2/Client.Aff.purs

```diff
@@ -0,0 +1,277 @@
+-- | Bindings to the *Node.js* HTTP/2 Client Core API.
+-- |
+-- | ## Client-side example
+-- |
+-- | Equivalent to
+-- | [*Node.js* HTTP/2 Core API __Client-side example__](https://nodejs.org/docs/latest/api/http2.html#client-side-example)
+-- |
+-- | ```
+-- | import Node.Stream.Aff (readAll)
+-- |
+-- | ca <- liftEffect $ Node.FS.Sync.readFile "localhost-cert.pem"
+-- |
+-- | either (liftEffect <<< Console.errorShow) pure =<< attempt do
+-- |   client <- connect
+-- |     (toOptions {ca})
+-- |     (URL.parse "https://localhost:8443")
+-- |
+-- |   stream <- request client
+-- |     (toOptions {})
+-- |     (toHeaders {":path": "/"})
+-- |
+-- |   headers <- waitResponse stream
+-- |   liftEffect $ for_ (headerKeys headers) \name ->
+-- |     Console.log $
+-- |       name <> ": " <> fromMaybe "" (headerString headers name)
+-- |
+-- |   body <- toStringUTF8 =<< (fst <$> readAll (toDuplex stream))
+-- |   liftEffect $ Console.log $ "\n" <> body
+-- |
+-- |   closeSession client
+-- | ```
+module Node.HTTP2.Client.Aff
```

Comment:
The file path for this file should changed:
```diff
-src/Node/Http2/Client.Aff.purs
+src/Node/Http2/Client/Aff.purs
```

Reply:
Really? I like this way. But if you insist, I'll make that change.

Reply:
I just went to make that change and now I remember why I don't like this: because then we end up with two files named `Aff.purs`. Which is not a good name for a file. And that convention gets worse as we scale up; if a repo has a lot of `Effect` binding modules with a lot of `Aff` API modules to go with them, then we just keep adding more files named `Aff.purs`.

Is there some tooling reason why we would prefer to accumulate `Aff.purs` files? Some tool which requires module prefixes to be subdirectories?

Reply:
There's nothing stopping someone from using qualified imports like
```purs
import Node.Http2.Client.Aff as Client.Aff
```

Reply:
> Really? I like this way. But if you insist, I'll make that change.

This just goes against convention with how everything else is done in core/contrib/web/node libs. I also wonder if this would break things in `purs-backend-es` if we did go with this approach and someone defined a module using the convention and exposed a similarly named function defined here.

Reply:
Ok done.